packages/gravity-ui-web/src/sass/01-tools/_layouts.scss

```diff
@@ -0,0 +1,53 @@
+// a mixin to add column based layout styles
+@mixin grav-l-column-layout($column-count: 2, $column-spacing: $grav-sp-m) {
+  display: flex;
+  flex-wrap: wrap;
+  padding-right: 0;
+  padding-left: 0;
+
+  > * {
+    width: calc((100% - #{(($column-count - 1) * 2) * $column-spacing}) / #{$column-count});
+    margin-right: $column-spacing;
+    margin-left: $column-spacing;
+  }
+
+  > *:nth-child(#{$column-count}n) {
```

Comment:
Perhaps add a comment for each of these `:nth-child` selectors to explain what they do. I know because we were chatting about it yesterday, but I'm not sure they'd be totally self-explanatory to others.

Perhaps something like:
```scss
// Remove outer margin from right-most item
> *:nth-child(#{$column-count}n) { ... }

// Remove outer margin from left-most item
> *:nth-child(#{$column-count}n - #{$column-count - 1}) { ... }

// Remove top margin from all items in first row
> *:nth-child(-n + #{$column-count}) { ... }
```